p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -3085,8 +3085,8 @@ tuple<bit<32>, bool> x = { 10, false };
 
 ## Operations on sets { #sec-set-exprs }
 
-Some P4 expressions denote sets of values (```set<T>```, for some type
-```T```; see Section [#sec-synth-types].1). These expressions can
+Some P4 expressions denote sets of values (```set<T>```, for some type ```T```;
+see Section [#sec-synth-types].1). These expressions can
```

Comment:
I did not add that, but it appears someone wanted to make a cross-ref to the first sub-section of #sec-synth-types, which is named "Set types".   I can try to make a named 'anchor' for that sub-section and see if Madoko likes it.